iFixit/react-commerce
# Add related products section

closes #710 

Adds [featured products section](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=6650%3A80646)

<img width="1056" alt="Screenshot 2022-09-27 at 16 25 08" src="https://user-images.githubusercontent.com/4640135/192553439-b48c1b65-c702-4c53-890c-649cf8aea378.png">

**Note**

- Workbench details are not implemented as we still lack a workbench implementation in react-commerce.
- I had to manually insert some data since metafields at this time are not populated

## QA

1. Open Vercel preview
2. Visit a [product page](https://react-commerce-git-add-related-products-section-ifixit.vercel.app/Products/apple-watch-38-mm-series-2-replacement-battery) that has featured products
3. Verify that the featured products section is rendered according to mockup
